po/de.po

```diff
@@ -0,0 +1,250 @@
+# German translations for PACKAGE package
+# German messages for PACKAGE.
+# Copyright (C) 2015 THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
```

Comment:
No, people seem to ignore these doc comments, anyway. messages.pot is auto-generated from the sources, <lang>.po is updated with new strings from messages.pot with msgmerge. The process is explained in TRANSLATION.md.